apps/test/unit/templates/projects/SectionProjectsListTest.js

```diff
@@ -65,6 +65,24 @@ function assertRowThumbnail(rowElement, thumbnailUrl) {
   expect(src).to.equal(thumbnailUrl);
 }
 
+/**
+ * @param {HTMLTableRowElement} rowElement HTML row element in the projects list table
+ * @param {string} projectName Expected project name
+ * @param {string} studentName Expected student name
+ * @param {string} appType Expected app type (App Lab, Game Lab, etc)
+ * @param {string} lastEdited Expected last edited date (Month DD, YYYY). Note that this
+ *   format is used only in unit tests due to incorrect date formatting in PhantomJS.
+ *   The desired date format which we will show in all browsers is MM/DD/YYYY.
+ */
+function assertRowContentsWithoutThumbnail(rowElement, projectName, studentName, appType, lastEdited) {
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.PROJECT_NAME].innerText).to.equal(projectName);
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.STUDENT_NAME].innerText).to.equal(studentName);
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.APP_TYPE].innerText).to.equal(appType);
+  // Temporarily comment out this line to make tests pass locally in Chrome
```

Comment:
In PhantomJS, `Date.toLocaleDateString()` has a non-standard output format of `December 29, 2016` instead of the standard `12/29/2016` output by Chrome. The Date class has options to explicitly specify the output format:
```
var options = {year: 'numeric', month: 'numeric', day: 'numeric' };
console.log(date.toLocaleDateString('en-US', options));
```
However PhantomJS doesn't respect these options either.